src/functions.lisp

```diff
@@ -292,3 +292,10 @@
   (channel amqp-channel-t)
   (id amqp-method-number-t)
   (decoded :pointer))
+
+(cffi:defcfun ("amqp_basic_qos" amqp-basic-qos) :pointer
+  (state amqp-connection-state-t)
+  (channel amqp-channel-t)
+  (prefetch-size uint32-t)
+  (prefetch-count uint64-t)
+  (global :boolean))
```

Comment:
Booleans in rabbitmq-c are declared as `amqp_boolean_t`, while the type of `:boolean` in CFFI is `int`. While in practice this should not be an issue, I would much rather see that the declarations exactly matches those in the rabbitmq-c header files. That means changing this to `amqp_boolean_t` and putting `(if global 1 0)` at the calling location.